### PR TITLE
[plugin.video.catchuptvandmore] 0.2.9

### PR DESCRIPTION
**Pull request by SylvainCecchetto**
### Description

[Version 0.2.9]
[Fix] Catch-up TV - KTO (FR) - Some programs
[Fix] Catch-up TV - MyCanal (FR) - Some programs
[Add] Catch-up TV - Piwi + and TéléToon + (FR)
[Fix] Catch-up TV - Rai (IT) - Check if program has contents
[Remove] Catch-up TV - J-One (FR) - No more contents in the website - Maybe will be readd in the future
[Fix] Catch-up TV - GameOne (FR)
[Add] Catch-up TV - Paramount Channel (FR)
[Move] Live TV - Paramount Channel (ES|IT) from WO to IT and ES
[Move] Catch-up TV - MTV (FR) from WO to FR
[Remove] Log_uploader for the exception 'No items found'
[Fix] Catch-up TV - Arte Concert (WO)
[Remove] Live TV - Virgin Radio (FR)
[Add] Website - Philharmoniedeparis.fr (FR)
[Fix] Catch-up TV - NPO (NL)
[Fix] Catch-up TV - RT (EN)
[Fix] Live TV - La7 (IT)
[Fix] Catch-up TV and Live TV - RMC Découverte (FR)
[Fix] Live TV - L'équipe TV (FR)

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by SylvainCecchetto** [comment]
Hi, you can find here our new PR in response to this answer https://github.com/xbmc/repo-plugins/pull/2186#issuecomment-515706726
Thank you ;-)

File: plugin.video.catchuptvandmore/setup.cfg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[flake8]
```
Comment (enen92) [inline plugin.video.catchuptvandmore/setup.cfg]: remove from submission

File: plugin.video.catchuptvandmore/resources/tools/mock_codequick.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```
Comment (enen92) [inline plugin.video.catchuptvandmore/resources/tools/mock_codequick.py]: not needed in the submission

**Comment by enen92** [comment]
Did you remove the module already?

**Comment by SylvainCecchetto** [comment]
I removed the files, but what do you mean about "module"? I don't understand :-/

**Comment by enen92** [comment]
For some reason the comment was not sent. Maybe because the submission has 491 files? :)
You need to remove the tzlocal module and submit it as a script.module for this addon to go in

**Comment by SylvainCecchetto** [comment]
Ah ok. You mean that we need to remove the tzlocal folder from our plugin and use `script.module.tzlocal` as an "import"?
But this module do not seems to be in official repo and we are not the developers of this module :-/

**Comment by enen92** [comment]
Yes, please do. We do not allow external modules to be bundled with add-ons. Set the original author, source repo and license in the add-on.xml of the module

**Comment by SylvainCecchetto** [comment]
That should be ok now ;-)

**Comment by enen92** [comment]
@SylvainCecchetto it will be merged next time the repo syncs (and travis is happy)

**Comment by SylvainCecchetto** [comment]
Cool! Thank you!

**Comment by Rechi** [comment]
@SylvainCecchetto this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Pull request merged**